<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>initWindow</class>
 <widget class="QWidget" name="initWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Water Analysis</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>엑셀 데이터</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="path_label">
          <property name="text">
           <string>Path:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="file_browse_edit"/>
        </item>
        <item>
         <widget class="QPushButton" name="file_browse_button">
          <property name="text">
           <string>Browse..</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="QTableView" name="excel_view"/>
        </item>
        <item>
         <widget class="QPushButton" name="excel_to_sql_button">
          <property name="text">
           <string>Extract as SQL</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="SettingGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="update_setting_button">
          <property name="text">
           <string>실시간 갱신 설정</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="advanced_setting_button">
          <property name="text">
           <string>Advanced...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="analysis_text">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="analysis_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>14</width>
         <height>28</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 255);</string>
       </property>
       <property name="text">
        <string>Analysis</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
